index.html

```diff
@@ -124,5 +124,30 @@
     <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
   </nav>
 
+  <div id="hiring-and-aearly-access-section">
+    <div id="join-hiring" class="card-subsection">
+      <h3> We're Hiring </h3>
+      <p> Help us redefine the calendar, reclaim our schedules, and make the time we spend together, better </p>
+      <a href="#" class="btn roles-button">See the roles</a>
+    </div>
+
+    <div id="picture-join">
+      <img id="flying-shoe" class="above-bg shake" src="images/joinbox-flying-shoe.png" alt="flying shoe">
+      <img id="pencil-small" class="above-bg shake" src="images/joinbox-pencil-small.png" alt="pencil small">
+      <img id="green-box" class="above-bg shake" src="images/joinbox-abstract-green-box.png" alt="abstract green box">
+      <img id="blue-ball" class="above-bg shake" src="images/joinbox-abstract-blue-ball.png" alt="abstract blue ball">
+      <img id="flying-febby" class="above-bg" src="images/joinbox-flying-febby.png" alt="flying febby">
+    </div>
+
+    <div id="early-access" class="card-subsection">
+      <h3> Join for early access </h3>
+      <p> We're gradually opening Ready—get early access to updates and be a part of building the next generation of calendars.</p>
+      <form id="form-access">
```

Comment:
where's the form action and method?

Reply:
Lo linkeo hacia una pagina ficticia o hacia la que manda la pagina original?
En un futuro vamos a implementar esto? o sea, hacer un servidor para que maneje esa request.

Reply:
> Lo linkeo hacia una pagina ficticia o hacia la que manda la pagina original? En un futuro vamos a implementar esto? o sea, hacer un servidor para que maneje esa request.

:v A una ficticia, si mandamos a la original les van a llegar weas y nos van a pillar con copyright y eso encima. 

Y no, no vamos a implementar el servidor para esto ahorita y no es necesario. Pero no es buen hábito dejar eso vacío. Luego que sí uses servidores y no tienes ni un relleno porqué se te olvidó, pues :v. Otra pull request más porqué se te olvidó poner esa wea en vez de haberte acostumbrado desde antes.

:v Pa servidores ya mejor vemos con el scrapper que vendrá luego. Aquí nomas se clona el sitio ahre.